internal/linker/test/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "devDependencies": {
+        "semver": "1.0.0"
```

Comment:
nit: maybe a comment that this version differs from the one in the root (and maybe also a comment in the root package.json to prevent us upgrading it to 1.0.0 by accident and losing the assertion)

Reply:
root package.json would be a downgrade as it is currently at 5.6.0. 

I added a negative test that will assert break if someone happens to downgrade root to < 5.0.0
```
  it('should get semver from root pacakge.json (which is currently 5.6.0) if the is no transitive in 1p dep',
     () => {
       const onepb = require('@rules_nodejs/onepb');
       const major = onepb.semver().major(onepb.semverVersion());
       expect(major).toBeGreaterThanOrEqual(5);
     });
```